applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -51,7 +51,7 @@ def _SetUpClassicEmbeddedNavierStokes(self, formulation_settings):
         self.condition_name = "NavierStokesWallCondition"
         self.level_set_type = formulation_settings["level_set_type"].GetString()
         self.element_integrates_in_time = True
-        self.element_has_nodal_properties = True
+        self.element_has_nodal_properties = False
```

Comment:
Do you still need thin embedded solver? All of them get these from elemental properties ryt?

Reply:
It is not needed but this member variable is used in the base Python solver. Although it can be created in the base constructor, I'd leave it in there for a while to avoid possible misunderstandings.